code/game/antagonist/station/traitor.dm

```diff
@@ -64,7 +64,7 @@
 	survive_objective = /datum/objective/survive
 
 /datum/antagonist/traitor/synth/can_become_antag(var/datum/mind/player)
```

Comment:
Traitor synth? I didn't know that existed yet, coding it is one of my tasks. Surprised to see there's some groundwork to start with